src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -163,22 +163,23 @@ public boolean isPinned()
     @Override
     public RestAction<Void> clearReactions()
     {
-        if (!isFromType(ChannelType.TEXT))
-            throw new IllegalStateException("Cannot clear reactions from a message in a Group or PrivateChannel.");
```

Comment:
This is likely intentional. Checks.check throws an IllegalArgumentException, which suggests that an illegal argument has been passed to the method. That is not the case here, so throwing an IllegalStateException is more appropriate. The same applies to the methods below.

Reply:
Yeah, I just recognized this 10min ago and I already started changing that back to throw an IllegalStateException. Before I didn't noticed that I was throwing the wrong exception now.

I will also add a note to the Docs of Message about that case.

Thanks for telling me though!